pkg/scalers/scaler.go

```diff
@@ -86,6 +86,24 @@ func GetFromAuthOrMeta(config *scalersconfig.ScalerConfig, field string) (string
 	return result, err
 }
 
+// GetFromAuthOrMetaOrEnv helps to get a field from Auth or Meta or Env sections
+func GetFromAuthOrMetaOrEnv(config *scalersconfig.ScalerConfig, field string, envField string) (string, error) {
+	var result string
+	var err error
+	switch {
+	case config.AuthParams[field] != "":
+		return config.AuthParams[field], err
+	case config.TriggerMetadata[field] != "":
+		return config.TriggerMetadata[field], err
+	case config.ResolvedEnv[config.TriggerMetadata[envField]] != "":
+		return config.TriggerMetadata[envField], err
+	}
+	if result == "" {
+		err = fmt.Errorf("%w: no %s or %s given", ErrScalerConfigMissingField, field, envField)
+	}
+	return result, err
+}
```

Comment:
Have added a new method  GetFromAuthOrMetaOrEnv so that any scaler making use of GetFromAuthOrMeta can have ENV variables read from it. Please let me know if you have any thoughts on the same